app/src/main/java/org/fossasia/openevent/app/utils/DateService.java

```diff
@@ -0,0 +1,53 @@
+package org.fossasia.openevent.app.utils;
+
+import org.fossasia.openevent.app.data.models.Event;
+
+import java.text.ParseException;
+import java.util.Date;
+
+public class DateService {
```

Comment:
Better we keep this (comparison and getting event status) in `Event.class` itself. That will be easier to manage as this class will become messy on further additions.